### DISCUSSION TITLE
How to resolve relative URL in schema/$ref?

### DISCUSSION THREAD

**Discussion by vrajpal-jhala** [jellydn/next-swagger-doc#779] [Q&A] [answered]
How can we resolve a relative URL provided in JSDoc with schema/$ref?

NextJS won't load JSON files directly so do we have to create route handlers to server those JSON files for each schema/$ref?

**Comment by jellydn** [comment]
Is that related with this one? @vrajpal-jhala https://github.com/jellydn/next-swagger-doc/discussions/800

**Reply by vrajpal-jhala** [reply, marked as answer]
Yes, What I did is added all routes that resolves a JSON schema and programmatically updated the path of the ref that resolves to those external schema. I believe the problem is that it does not resolves the path correctly to the external schema like described in the mentioned issue.
